src/pages/fresh_install/FreshInstallPage.css

```
@@ -0,0 +1,100 @@
+.DivButtons {
+    margin-top: 3vh;
+}
+
+.ButtonCelsius {
+    margin-right: 2rem;
+}
+
+.Button {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.9);
+    display: flex;
+    align-items: center;
+}
+
+.ButtonActive {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.35);
+    display: flex;
+    align-items: center;
+}
+
+.DivContent {
```

I don't think adding the component type on a class name is a good idea. What if we need to use it in another component that is not `<div>`? The same if we need to change the tag type to another one.

```suggestion
.Content {
```